transport/src/main/java/io/netty/bootstrap/Bootstrap.java

```diff
@@ -58,13 +60,15 @@
     private volatile AddressResolverGroup<SocketAddress> resolver =
             (AddressResolverGroup<SocketAddress>) DEFAULT_RESOLVER;
     private volatile SocketAddress remoteAddress;
+    volatile ChannelFactory<? extends Channel> channelFactory;
```

Comment:
wouldn't a bootstrapbuilder make more sense here to avoid the volatile fields (=> final)?

Reply:
@johnou we can consider this as a followup. This is not related to this PR.